frontendFile/src/main/java/com/virtuslab/gitmachete/frontend/file/highlighting/MacheteSyntaxHighlighter.java

```diff
@@ -0,0 +1,51 @@
+package com.virtuslab.gitmachete.frontend.file.highlighting;
+
+import static com.intellij.openapi.editor.colors.TextAttributesKey.createTextAttributesKey;
+import static io.vavr.API.$;
+import static io.vavr.API.Case;
+import static io.vavr.API.Match;
+
+import com.intellij.lexer.Lexer;
+import com.intellij.openapi.editor.DefaultLanguageHighlighterColors;
+import com.intellij.openapi.editor.HighlighterColors;
+import com.intellij.openapi.editor.colors.TextAttributesKey;
+import com.intellij.openapi.fileTypes.SyntaxHighlighterBase;
+import com.intellij.psi.TokenType;
+import com.intellij.psi.tree.IElementType;
+import org.checkerframework.checker.guieffect.qual.UI;
+
+import com.virtuslab.gitmachete.frontend.file.grammar.MacheteLexerAdapter;
+import com.virtuslab.gitmachete.frontend.file.grammar.MacheteTypes;
+
+@UI
+public class MacheteSyntaxHighlighter extends SyntaxHighlighterBase {
+  public static final TextAttributesKey BAD_CHAR = createTextAttributesKey("MACHETE_BAD_CHARACTER",
```

Comment:
Here we can customize default colors for our machete elements

Reply:
Nit: `BAD_CHAR` vs `BAD_CHARACTER`

Reply:
Changed to `BAD_CHARACTER`